### ISSUE TITLE
Import from other tools

### ISSUE THREAD

**Issue by oliviertassinari** [mui/mui-toolpad#1134]
### Duplicates

- [X] I have searched the existing issues

### Latest version

- [X] I have tested the latest version

### Summary 💡

Make it possible to migrate from other no-code/low-code.

### Examples 🌈

- https://help.figma.com/hc/en-us/articles/360040514273-Import-Sketch-files
- https://docs.nocodb.com/setup-and-usages/import-airtable-to-sql-database-within-a-minute-for-free

**Comment by prakhargupta1** [comment]
Is it possible to write a retool JSON parser that Toolpad can read and show as a app?

**Comment by Janpot** [comment]
> Is it possible to write a retool JSON parser that Toolpad can read and show as a app?

I don't think there's enough feature parity to make this useful. Besides, their data format is un(der)documented, and probably frequently changing, we'd be chasing a moving target.

Personally I don't think this is a realistic feature request. At least not until we have reasonable feature parity and are prepared to **permanently** dedicate a team member to reverse engineer changes to the retool format.

**Comment by bytasv** [comment]
I agree with Jan. I also think that such feature (if we decide to do it) should appear only when we would like to "convert" retool (or other tools) users to Toolpad users, something I imagine happening way later in development, once we have solid userbase already
